stl/src/format.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+

```

Comment:
If this is being injected into the import lib, it needs the Scary Import Lib Comment:

https://github.com/microsoft/STL/blob/4ee3f91e06956ed2d8da4adcb8fcbf6bd75dc3b9/stl/src/filesystem.cpp#L7-L10